assets/src/components/amp-support/plugins.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ListItems } from '../list-items';
+
+/**
+ * To render plugins information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Plugins data.
+ * @return {JSX.Element|null} HTML markup for plugins data.
+ */
+export function Plugins( { data: plugins } ) {
+	if ( 'object' !== typeof plugins ) {
+		return null;
+	}
+
+	plugins = Object.values( plugins );
```

Comment:
How about defining here the the actual list `items` array (instead of in the `items` property in the `return` statement):

```suggestion
	const items = Object.values( plugins ).map( ( item ) => ( {
		value: `${ item.name }${ item.version ? ` (${ item.version })` : '' }`,
	} ) );
```